recipes/sparse_dot_mkl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sparse_dot_mkl" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 352341c5cf739f72f49ba3df972f5fe99b08b7d2dfebe14367cfa857d0f4e9e0
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
Can this recipe be noarch python?

Reply:
Yes; it specifically is for noarch. The conda-forge MKL dependency is broken for windows though. As long as I can leave a win skip or just override the build I redo the recipe as noarch.

Reply:
noarch recipes cannot use selectors. However, you do not need to skip windows. If the solver cannot find mkl for windows, it simply won't install the package. So remove the skip statement and add `noarch: python`

Reply:
I am perfectly happy to merge the current `noarch` recipe that fails windows build check.